target/scripts/update-check.sh

```diff
@@ -3,8 +3,8 @@
 # shellcheck source=./helpers/log.sh
 source /usr/local/bin/helpers/log.sh
 
-VERSION=$(</VERSION)
-VERSION_URL='https://raw.githubusercontent.com/docker-mailserver/docker-mailserver/master/VERSION'
+VERSION="${DMS_VERSION#v}"
```

Comment:
What is `DMS_VERSION`?

Reply:
Good catch!

That was me being wishy/washy over `DMS_VERSION` or `DMS_RELEASE` and forgetting to update this one 😅

Reply:
```suggestion
VERSION="${DMS_RELEASE#v}"
```